src/components/Heading/types.ts

```diff
@@ -12,7 +12,7 @@ export const sizes = {
   LG: "lg",
   XL: "xl",
   XXL: "xxl",
-};
+} as const;
```

Comment:
Without this the size was a string. So when I was using it in the profile

```
<Heading size="sm" />
```

It wasn't returning an error but started to after upgrading Typescript.